### DISCUSSION TITLE
How to consume single razor web component in XAML?

### DISCUSSION THREAD

**Discussion by Sebosek** [dotnet/maui#15165] [Q&A]
I'm trying to workaround this nasty bug that completely blocks me (https://github.com/dotnet/maui/issues/15147), I would like to try to create a custom button component using the HTML + CSS (Razer Class Library), but so far I've only seen whole screens created as Razor Components, I would like to use only a single tiny component, is it even possible?
Would the data bindings work for this component? Has anyone done it before?

**Comment by davidortinau** [comment]
Check out this page that uses a single signature razor component in a XAML page. https://github.com/dotnet/maui-samples/blob/main/7.0/Apps/PointOfSale/src/PointOfSale/Pages/Handheld/SignaturePage.xaml#L23

**Reply by Xyncgas** [reply]
Yeah use a BlazorWebView directly don't think too much
